Sources/SourceKitLSP/Swift/SwiftLanguageService.swift

```diff
@@ -917,7 +920,7 @@ extension SwiftLanguageService {
 
   func retrieveQuickFixCodeActions(_ params: CodeActionRequest) async throws -> [CodeAction] {
     let snapshot = try await self.latestSnapshot(for: params.textDocument.uri)
-    let buildSettings = await self.buildSettings(for: params.textDocument.uri)
+    let buildSettings = await self.buildSettings(for: params.textDocument.uri, fallbackAfterTimeout: true)
```

Comment:
Diagnostics is false (though clangd publish diagnostics is true), do we want fixes to be true?

Reply:
Yes, because Fix-Its are delivered as part of code actions (which include Fix-Its + refactorings). Using a fallback here allows us to deliver syntactic refactorings even if we don’t have build settings.